app/models/concerns/exportable.rb

```diff
@@ -127,11 +127,11 @@ def last_call_timestamp
   end
 
   def call_count
-    calls.count
+    calls.size
```

Comment:
Fun fact: `size` will use the cached query, and `count` will always run a `count(*)` query. So this winds up saving us a lot of queries.